addons/gdUnit3/GdUnitCmdTool.gd

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env -S godot -s
+extends SceneTree
```

Comment:
Guess I have to run this `addons/gdUnit3/GdUnitCmdTool.gd`

Reply:
yes have a look at https://github.com/MikeSchulze/gdUnit3/wiki/Command-Line-Tool for details